### DISCUSSION TITLE
Need Help With Using Sandbox to Test Manifests

### DISCUSSION THREAD

**Discussion by Rolling2405** [microsoft/winget-pkgs#27544] [Q&A] [answered]
How do I use Windows Sandbox to test a Manifest?

**Comment by jedieaston** [comment, marked as answer]
The easiest way is via the SandboxTest.ps1 script in the Tools directory.

From PowerShell (presuming your working directory is the root of the repo):

`.\Tools\SandboxTest.ps1 <path to your manifest's folder>` will run it with the default options.

**Comment by Rolling2405** [comment]
I think this is my problem 

> From PowerShell (presuming your working directory is the root of the repo):

How do I do this?

**Comment by jedieaston** [comment]
If you have the repo open in Windows Explorer, go to the File menu and click “Open PowerShell window”. If you’re in VSCode, it should be the folder that opens when you open a terminal.

**Comment by vedantmgoyal9** [comment]
@Rolling2405 Is your issue fixed?
